lib/components/inputs/TextBox/TextBox.tsx

```diff
@@ -140,6 +155,11 @@ const TextBox = React.forwardRef(function TextBox(props: TextBoxProps, ref) {
             {Utils.getPasswordIcon(showPassword, toggleShowPassword)}
           </span>
         )}
+        {isLoadingValue && (
+          <span className='loader-wrapper'>
+            <InputLoader />
+          </span>
+        )}
```

Comment:
Why don't we use the `loader-wrapper` as part of the component itself?

Reply:
because we only need it in TextBox (not in Select and CustomizableSelect where it's also used)